cypher/cypher-docs/src/docs/dev/ql/administration/security/grant-deny-syntax.asciidoc

```diff
@@ -9,23 +9,23 @@ The syntax is:
 | Command | Description
 
 | [source, cypher]
-GRANT graph-privilege ON GRAPH[S] {name \| *} [entity] TO roles
+GRANT graph-privilege ON GRAPH[S] {name \| *} [entity] TO role[, ...]
 | Grant a privilege to one or multiple roles
 
 | [source, cypher]
-DENY graph-privilege ON GRAPH[S] {name \| *} [entity] TO roles
+DENY graph-privilege ON GRAPH[S] {name \| *} [entity] TO role[, ...]
 | Deny a privilege to one or multiple roles
 
 | [source, cypher]
-REVOKE GRANT graph-privilege ON GRAPH[S] {name \| *} [entity] FROM roles
+REVOKE GRANT graph-privilege ON GRAPH[S] {name \| *} [entity] FROM role[, ...]
 | Revoke a granted privilege from one or multiple roles
 
 | [source, cypher]
-REVOKE DENY graph-privilege ON GRAPH[S] {name \| *} [entity] FROM roles
+REVOKE DENY graph-privilege ON GRAPH[S] {name \| *} [entity] FROM role[, ...]
 | Revoke a denied privilege from one or multiple roles
 
 | [source, cypher]
-REVOKE graph-privilege ON GRAPH[S] {name \| *} [entity] FROM roles
+REVOKE graph-privilege ON GRAPH[S] {name \| *} [entity] FROM role[, ...]
 | Revoke a granted or denied privilege from one or multiple roles
 |===
 
```

Comment:
Further down in this file is `{props}` in the privilege list (both for READ and MATCH), those should have been updated as well

Reply:
there's also a `ELEMENTS+*+` which can be updated at the same time (shouldn't have `+`s)

Reply:
(if the `+` are removed confirm that the text doesn't get bold)

Reply:
@Lojjs We should probably decide how we want to do in these lists, just adding `[, ...]` is not fully correct. A more accurate version would be `"{" {* | prop[, ...]} "}"`, which is very chunky.

We also have different handling of the user list for transaction privileges, the label list for label privileges and these property lists for read/match/set property and merge.

I think the easiest is to not include the users/labels/properties in these lists and just have the privilege names (this is what `MERGE` to)

See also my other comment about this, https://github.com/neo4j/neo4j-documentation/pull/747#discussion_r528557043